wpilibNewCommands/src/test/native/cpp/frc2/command/WaitCommandTest.cpp

```diff
@@ -27,3 +27,60 @@ TEST_F(WaitCommandTest, WaitCommandSchedule) {
 
   frc::sim::ResumeTiming();
 }
+
+TEST_F(WaitCommandTest, WaitCommandScheduleLambda) {
+  frc::sim::PauseTiming();
+
+  CommandScheduler scheduler = GetScheduler();
+
+  WaitCommand command([] { return 100_ms; });
+
+  scheduler.Schedule(&command);
+  scheduler.Run();
+  EXPECT_TRUE(scheduler.IsScheduled(&command));
+  frc::sim::StepTiming(110_ms);
+  scheduler.Run();
+  EXPECT_FALSE(scheduler.IsScheduled(&command));
+
+  frc::sim::ResumeTiming();
+}
+
+TEST_F(WaitCommandTest, WaitCommandScheduleNonStaticLambda) {
+  int calls = 1;
+  frc::sim::PauseTiming();
+
+  CommandScheduler scheduler = GetScheduler();
+
+  WaitCommand command([calls] {
```

Comment:
```suggestion
  WaitCommand command([&calls] {
```
I think it's capturing by value, and thus not `calls=1` all the time. 
Capture by ref instead.